src/ssd/resize_indicator.c

```diff
@@ -114,8 +115,17 @@ resize_indicator_update(struct view *view)
 
 	char text[32]; /* 12345 x 12345 would be 13 chars + 1 null byte */
 	if (view->server->input_mode == LAB_INPUT_STATE_RESIZE) {
-		snprintf(text, sizeof(text), "%d x %d",
-			view->current.width, view->current.height);
+		int width = view->current.width;
+		int height = view->current.height;
+		if (view->impl->fill_size_hints) {
+			struct wlr_box hints;
```

Comment:
Should we initialise hints to zero here (={0})

Reply:
Not strictly necessary as `xwayland_view_fill_size_hints()` always sets `width` and `height`.
However, due to the addition of the `Nonpixel` variant I had to restructure some stuff and its now done with a helper which initializes the box with `{ 0 }`.